test/integration_test.go

```diff
@@ -227,59 +266,86 @@ func configureRBAC(t *testing.T, coreClient client.Interface) {
 	require.NoError(t, err)
 
 	_, err = clientRBAC.RoleBindings(core.NamespaceDefault).Create(&rbac.RoleBinding{
-		ObjectMeta: meta.ObjectMeta{Name: "alice-can-create-configmaps"},
+		ObjectMeta: meta.ObjectMeta{
+			Name:   "alice-can-create-configmaps",
+			Labels: commonLabels,
+		},
 		RoleRef: rbac.RoleRef{
 			Name: "create-configmaps",
 			Kind: cmd.RoleKind,
 		},
 		Subjects: []rbac.Subject{
-			{Kind: rbac.UserKind, Name: "Alice"},
+			{
+				Kind: rbac.UserKind,
+				Name: "Alice",
+			},
 		},
 	})
 	require.NoError(t, err)
 
 	_, err = clientRBAC.RoleBindings(core.NamespaceDefault).Create(&rbac.RoleBinding{
-		ObjectMeta: meta.ObjectMeta{Name: "rory-can-create-configmaps"},
+		ObjectMeta: meta.ObjectMeta{
+			Name:   "rory-can-create-configmaps",
+			Labels: commonLabels,
+		},
 		RoleRef: rbac.RoleRef{
 			Name: "create-configmaps",
 			Kind: cmd.ClusterRoleKind,
 		},
 		Subjects: []rbac.Subject{
-			{Kind: rbac.UserKind, Name: "Rory"},
+			{
+				Kind: rbac.UserKind,
+				Name: "Rory",
+			},
 		},
 	})
 	require.NoError(t, err)
 
 	_, err = clientRBAC.Roles(core.NamespaceDefault).Create(&rbac.Role{
-		ObjectMeta: meta.ObjectMeta{Name: "scale-workloads"},
+		ObjectMeta: meta.ObjectMeta{
+			Name:   "scale-workloads",
+			Labels: commonLabels,
+		},
 		Rules: []rbac.PolicyRule{
 			{
-				APIGroups: []string{"extensions"},
+				APIGroups: []string{"apps"}, // TODO For old clusters it extensions / never are apps
```

Comment:
nit: I think there is a typo in this comment, is it something like "For old clusters it's extensions, newer are apps"?

Reply:
Well spotted. I was typing toooooo fast here :)